benchmark/scenarios/write_request_load.py

```diff
@@ -0,0 +1,349 @@
+# Copyright 2015 ClusterHQ Inc.  See LICENSE file for details.
+"""
+Read request load scenario for the control service benchmarks.
+"""
+from collections import deque
+from itertools import repeat
+
+from zope.interface import implementer
+from eliot import start_action, write_failure, Message
+from eliot.twisted import DeferredContext
+
+from twisted.internet.defer import CancelledError, Deferred
+from twisted.internet.task import LoopingCall
+
+from flocker.common import loop_until, timeout
+
+from .._interfaces import IScenario
+
+DEFAULT_SAMPLE_SIZE = 5
+
+
+class WRateMeasurer(object):
+    # XXX make it a common class for Read and Write scenarios.
+    # Note that the docstrings are not up-to-date because they are being
+    # updated in the Read scenario
```

Comment:
Add an issue in the Benchmarking epic to refactor read and write load for more sharing, and add the issue number here, and anywhere else it seems relevant.